README.md

```diff
@@ -132,6 +132,22 @@ Extensive Example `flake.nix`:
 }
 ```
 
+Initializing a dream2nix from a nixpkgs set:
+```nix
+let
+  system = "x86_64-linux";
+  # in this case, 'nixpkgs' is a nixpkgs flake
+  pkgs = nixpkgs.legacyPackages.${system};
+  # 'dream2nix' is a dream2nix flake
+  d2n = dream2nix.lib.init {inherit pkgs;};
+in
+  d2n.realizeProjects {
```

Comment:
Should this be accessible, if `makeOutputs` is `realizedProjects // apps.resolveImpure`? Is there a case where you'd use realizeProjects and not makeOutputs?

Reply:
I think we should keep it just to offer more options. We may add more stuff to `makeOutputs` or do some changes with the outputs, so it's probably better to offer a way to only get the builder outputs and nothing else.